spec/resource/collection_spec.rb

```diff
@@ -230,19 +230,19 @@
 
         expect(directory.groups.limit(30).count).to eq(100)
 
-        expect(directory.groups.limit(30).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(30).count).to eq(70)
 
-        expect(directory.groups.limit(30).offset(30).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).offset(30).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(60).count).to eq(40)
 
-        expect(directory.groups.limit(30).offset(60).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).offset(60).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(90).count).to eq(10)
 
-        expect(directory.groups.limit(30).offset(90).current_page.items.count).to eq(10)
+        expect(directory.groups.limit(30).offset(90).size).to eq(10)
```

Comment:
Are these tests still passing after having removed the size from the `Collection` class? Seems like the tests would be calling `current_page.size` unless I'm missing something.